the_mines/get_html.py

```diff
@@ -0,0 +1,47 @@
+from argparse import ArgumentParser
+import requests
+import shutil
+
+
+def get_html():
+    parser = ArgumentParser()
+    parser.add_argument(
+        "--matchday", action="store", required=True, help="ex. 8",
+    )
+    parser.add_argument(
+        "--season", action="store", required=True, help="ex. 2019/2020",
+    )
+    args = parser.parse_args()
+
+    matchday = f"{args.matchday}"
+    season = f"{args.season}"
+
+    url = "https://www.fussballdaten.de/bundesliga/" + season[5:9] + "/" + matchday
```

Comment:
It would be helpful to have a comment here explaining what the `season[5:9]` is doing.